static/js/documentation.js

```diff
@@ -81,6 +81,26 @@ $(function() {
     return false
   })
 
+  // Toggle between expanding and collapsing all categories
+  $('.expand-all a').click(function() {
+    $(this).text() == '+ Expand All' ? $(this).text('- Collapse All') : $(this).text('+ Expand All')
+    $('.js-toggle-list .js-topic').each(function() {
+      var li = $(this).find('.js-guides li')
+      if($('.expand-all a').text() == '- Collapse All') {
+        if(li.css('display') == 'none') {
```

Comment:
Can we use a class instead?

Reply:
nit: add space after `if`

Reply:
I switched to checking the class on the expand button: https://github.com/cerner/fhir.cerner.com/pull/431/commits/0baf63e3ea3633fa55a34e5b728142f6662b959c#diff-46bd8686b23677fb5c466a0f71fa66fdR92